torchmultimodal/modules/encoders/clip_vit_encoder.py

```diff
@@ -6,11 +6,7 @@
 
```

Comment:
oops why is this file still around. will remove

Reply:
do you mean I should remove this file completely? Should it be replaced by `VisionTransform` on Rafi's PR?